sequencer/src/block2.rs

```diff
@@ -155,6 +155,17 @@ impl BlockPayload {
     }
 }
 
+// Read TxTableEntry::byte_len() bytes from `table_bytes` starting at `offset`.
+// if `table_bytes` has too few bytes at this `offset` then pad with zero.
+// Parse these bytes into a `TxTableEntry` and return.
+fn get_table_len(table_bytes: &[u8], offset: usize) -> TxTableEntry {
+    let tx_table_len_range =
+        offset..std::cmp::min(TxTableEntry::byte_len(), table_bytes.len()) + offset; // refactor to table_len_range()?
```

Comment:
Should this be `min(offset + TxTableEntry::byte_len(), table_bytes.len())`?

Reply:
fixed in https://github.com/EspressoSystems/espresso-sequencer/pull/998/commits/fbd14fad75e9d05ab06fa3840db1923059e6b536
Amazing what mistakes can slip by two people looking at the code. 😬